frontend/test/metabase-db/mysql/custom-column.cy.spec.js

```diff
@@ -0,0 +1,60 @@
+import {
+  restore,
+  signInAsAdmin,
+  addMySQLDatabase,
+  withDatabase,
+} from "__support__/cypress";
+
+const MYSQL_DB_NAME = "QA MySQL8";
+
+// NOTE: skipping the whole describe block because #12445 is the only test so far
+describe.skip("mysql > user > question > custom column", () => {
+  beforeEach(() => {
+    restore();
+    signInAsAdmin();
+    addMySQLDatabase(MYSQL_DB_NAME);
+  });
+
+  // TODO: When you unskip this test, unskip the describe block as well
+  it.skip("should correctly apply substring for a custom column (metabase#12445)", () => {
+    const CC_NAME = "Abbr";
+
+    withDatabase(2, ({ PEOPLE, PEOPLE_ID }) => {
+      cy.log(
+        "**--1. Create a question with `Source` column and abbreviated CC--**",
+      );
+      cy.request("POST", "/api/card", {
+        name: "12445",
+        dataset_query: {
+          type: "query",
+          query: {
+            "source-query": {
+              "source-table": PEOPLE_ID,
+              breakout: [["field-id", PEOPLE.SOURCE]],
+            },
+            expressions: {
+              [CC_NAME]: [
+                "substring",
+                ["field-literal", "SOURCE", "type/Text"],
```

Comment:
Perhaps I'm misreading something, or I don't understand what the Notebook editor looks like. This is what I think you are trying to do?
![image](https://user-images.githubusercontent.com/1447303/101685591-3c714c80-3a68-11eb-8b78-3963c9fdc778.png)
```suggestion
            "source-table": PEOPLE_ID,
            breakout: [["expression", CC_NAME]],
            expressions: {
              [CC_NAME]: [
                "substring",
                ["field-id", PEOPLE.SOURCE],
```

Reply:
Yes, I was trying to do that, although it doesn't make much sense I guess. 🤨

Reply:
> Perhaps I'm misreading something, or I don't understand what the Notebook editor looks like. This is what I think you are trying to do?

Actually, no. I was going for:
![image](https://user-images.githubusercontent.com/31325167/101691157-92e18980-3a6e-11eb-9053-286761bff262.png)

Which results in:
![image](https://user-images.githubusercontent.com/31325167/101691194-9ffe7880-3a6e-11eb-942e-880e388e1dd1.png)


But your solution is definitely simpler.